src/modules/Modal/Modal.js

```diff
@@ -155,13 +158,16 @@ class Modal extends Component {
   static Description = ModalDescription
   static Actions = ModalActions
 
+  // expose for testing
+  static isBrowser = isBrowser
```

Comment:
Is there a reason to not import `isBrowser` in tests instead?

Reply:
Oh, I see.  I'd like to have a way to spoof `isBrowser` that is not dependant on the Modal.  One idea is to convert isBrowser to a function.  This way, we can set a flag to override it:

```js
const isBrowser = () => !_.isNil(isBrowser, 'override')
  ? isBrowser.override
  : hasDocument && hasWindow

export default isBrowser
```

Then we can do something similar to what is being done now, except with isBrowser rather than Modal:

```jsx
before(() => {
  isBrowser.override = false
})

after(() => {
  isBrowser.override = null
})
```